eng/targets/Settings.props

```diff
@@ -71,7 +71,7 @@
       These may be overridden by projects that need to be skipped.
     -->
     <TestTargetFrameworks Condition="'$(TestRuntime)' == 'Mono'">net46;net472</TestTargetFrameworks>
-    <TestTargetFrameworks Condition="'$(TestRuntime)' == 'Core'">net6.0;net7.0</TestTargetFrameworks>
+    <TestTargetFrameworks Condition="'$(TestRuntime)' == 'Core'">net6.0;net6.0-windows;net7.0</TestTargetFrameworks>
```

Comment:
What does this property actually do?

Reply:
No idea. I opened this as a test. I probably should have just grep'd over code.